src/App.vue

```diff
@@ -8,21 +8,40 @@
 </template>
 
 <script>
+import apiMoviesService from "../src/services/api/apiMoviesService";
 import TheHeader from "./components/TheHeader.vue";
 
 export default {
   name: "App",
   components: {
     TheHeader,
   },
+
+  methods: {
+    async getMovies() {
+      try {
+        const movies = await apiMoviesService.getMovieList();
+        this.$store.commit("setMovies", movies);
+      } catch (err) {
+        console.log(err);
+      }
+    },
```

Comment:
If you already keep the response in the `store`, it is probably a good idea to move also this logic to the store and make it an `action`.

Then all the logic related to fetching and managing the state of the list of movies would be in the store, which would be easy to find and easier to maintain.

Reply:
Love this one! Thank you ❤️